blueocean-dashboard/src/main/js/creation/credentials/CreateCredentialDialog.jsx

```diff
@@ -198,7 +198,7 @@ export class CreateCredentialDialog extends React.Component {
         const disabled = this.state.creationPending;
 
         const buttons = [
-            <button className="button-create-credental" disabled={disabled} onClick={() => this._onCreateClick()}>
```

Comment:
This change is tied to corresponding changes in ATH.

Reply:
👍  was looking for that one